lib/core/context-menu/context-menu-overlay.service.ts

```diff
@@ -102,26 +102,33 @@
         };
 
         const positionStrategy = this.overlay.position()
-            .connectedTo(
-                new ElementRef(fakeElement),
-                { originX: 'start', originY: 'bottom' },
-                { overlayX: 'start', overlayY: 'top' })
-            .withFallbackPosition(
-                { originX: 'start', originY: 'top' },
-                { overlayX: 'start', overlayY: 'bottom' })
-            .withFallbackPosition(
-                { originX: 'end', originY: 'top' },
-                { overlayX: 'start', overlayY: 'top' })
-            .withFallbackPosition(
-                { originX: 'start', originY: 'top' },
-                { overlayX: 'end', overlayY: 'top' })
-            .withFallbackPosition(
-                { originX: 'end', originY: 'center' },
-                { overlayX: 'start', overlayY: 'center' })
-            .withFallbackPosition(
-                { originX: 'start', originY: 'center' },
-                { overlayX: 'end', overlayY: 'center' }
-            );
+            .flexibleConnectedTo(new ElementRef(fakeElement))
+            .withPositions([
+                {
+                    originX: 'start', originY: 'bottom' ,
+                    overlayX: 'start', overlayY: 'top'
+                },
+                {
+                    originX: 'start', originY: 'top',
+                    overlayX: 'start', overlayY: 'bottom'
+                },
+                {
+                    originX: 'end', originY: 'top',
+                    overlayX: 'start', overlayY: 'top'
+                },
+                {
+                    originX: 'start', originY: 'top',
+                    overlayX: 'end', overlayY: 'top'
+                },
+                {
+                    originX: 'end', originY: 'center',
+                    overlayX: 'start', overlayY: 'center'
+                },
+                {
+                    originX: 'start', originY: 'center',
+                    overlayX: 'end', overlayY: 'center'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Alfresco/alfresco-ng2-components/pullRequest?prid=8983907)
```suggestion
                    overlayX: "end", overlayY: 'center'
```